docs/Makefile

```diff
@@ -0,0 +1,155 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
if we're using `sphinx-apidoc` to generate the docs, do we need this Makefile?

Reply:
The `sphinx-apidoc` generates the .rst files, that are used in this Makefile when we call `make html` (look at line 34 of the Makefile above).

Reply:
I've moved all the autogenerated `.rst` files to `source/_rst`. Now the directory is cleaned up every time new docs are generated (also added to .gitignore). This addresses your concern about deprecated modules (even though the rst files are just directives, so a new html wouldn't be created but the deprecated `.rst` file would stay there).

I would prefer to keep the two `Makefile`s, if you don't have a strong motivation behind merging them. Sphix's internal variables would make `twilio-python/Makefile` too crowded.